wrapper/telemetry-wrapper.js

```diff
@@ -353,7 +352,7 @@ window.TelemetryWrapper.go = function (params, element) {
     var percentileLabel = ' - medians'; // i18n?
     var valuesArePercent = false;
     if (kind == 'enumerated' || kind == 'boolean' || kind == 'flag') {
-      const BUCKET_INDEX_FOR_ENUMERATED = 0;
+      const BUCKET_INDEX_FOR_ENUMERATED = params.evoBucketIndex > 0 ? params.evoBucketIndex : 0;
```

Comment:
CONSTANT_NAMING_STYLE is something usually reserved for compile-time constants and static configurations. Since we're now configuring this at runtime I think it's worth reworking this to perhaps be called `bucketIndex` or something else written in camelCase so we don't lose that distinction.